Zend/zend_API.c

```diff
@@ -3483,6 +3492,10 @@ static zend_always_inline bool zend_is_callable_check_func(int check_flags, zval
 			if (error) zend_spprintf(error, 0, "class %s is not a subclass of %s", ZSTR_VAL(ce_org->name), ZSTR_VAL(fcc->calling_scope->name));
 			return 0;
 		}
+		if (ce_org && error) {
+			zend_error(E_DEPRECATED,
+				"Callables of the form [\"C1\", \"C2::method\"] are deprecated");
```

Comment:
```suggestion
				"Callables of the form [\"%s\", \"%s\"] are deprecated", ZSTR_VAL(ce_org->name), ZSTR_VAL(Z_STR_P(callable)));
```

Do you think it will be better if we showed the exact method names here? It is likely that these callables are called with variable objects, so showing the exact name of the object can help figure out where it comes from.